operator/v1/doc.go

```diff
@@ -2,5 +2,6 @@
 // +k8s:defaulter-gen=TypeMeta
 // +k8s:openapi-gen=true
 
+// +kubebuilder:validation:Optional
```

Comment:
really capital case?

Reply:
Yup, see https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/package.go#L27-L28

They could have been defined as lowercase but it's consistent with the rest of the `kubebuilder:validation` markers (defined here https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/validation.go#L34-L61), which are parsed based on their type name (via `reflect.TypeOf(obj).Name()` here https://github.com/kubernetes-sigs/controller-tools/blob/master/pkg/crd/markers/register.go#L33) which leads to capitalized markers

Reply:
how ugly